src/client/package.json

```diff
@@ -41,6 +42,7 @@
     "openfin:start:launcher": "concurrently \"npm:openfin:dev\" \"npm:openfin:run:launcher\"",
     "openfin:build": "cross-env TARGET=openfin vite build",
     "openfin:serve": "cross-env TARGET=openfin vite preview",
+    "openfin:test":"playwright test --project=openfin",
```

Comment:
I think I would prefer `test:e2e:openfin` and then `test:e2e:web` above .. and any variations on that as we add more cases, like say `test:web:fx` and so on... we could even go with just `e2e:web` etc?

Reply:
Sounds good